### ISSUE TITLE
Error on https://armorgames.com/play/5885/castaway

### ISSUE THREAD

**Issue by ahmadn91** [ruffle-rs/ruffle#8016]
# Error Info
Error name: Error
Error message: panicked at 'already borrowed: BorrowMutError', core/src/bitmap/bitmap_data.rs:1001:22
Error stack:
```
Error: panicked at 'already borrowed: BorrowMutError', core/src/bitmap/bitmap_data.rs:1001:22
    at W.n.wbg.__wbg_new_df6e6ab7a65c4c4d (https://agi.armorgames.com/ruffle/core.ruffle.62a5b45e6d4e022d7d4b.js:1:5314)
    at std::sync::once::Once::call_once::{{closure}}::hc7e1e9ccae1dc751 (wasm://wasm/0171db26:wasm-function[946]:0x272e6a)
    at std::sync::once::Once::call_inner::h00971925aa1adad3 (wasm://wasm/0171db26:wasm-function[1397]:0x2f6e0e)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::heaafb58166fe27c7 (wasm://wasm/0171db26:wasm-function[4859]:0x453959)
    at std::panicking::rust_panic_with_hook::h84feca33bd4bd229 (wasm://wasm/0171db26:wasm-function[3612]:0x41caa7)
    at core::panicking::panic_fmt::hc171d095bc4a492d (wasm://wasm/0171db26:wasm-function[5046]:0x45a07e)
    at core::result::unwrap_failed::h68ab818eb89182b6 (wasm://wasm/0171db26:wasm-function[4173]:0x43ec00)
    at ruffle_core::bitmap::bitmap_data::BitmapData::draw::hc7b7f287487460ed (wasm://wasm/0171db26:wasm-function[682]:0x1ff41e)
    at ruffle_core::avm1::globals::bitmap_data::draw::hb54682cb84e6552d (wasm://wasm/0171db26:wasm-function[677]:0x1fd23d)
    at ruffle_core::avm1::function::Executable::exec::he45fc3033712f2de (wasm://wasm/0171db26:wasm-function[353]:0xd85d9)
```

# Player Info
Allows script access: true
Renderer: WebGL

# Page Info
Page URL: https://armorgames.com/play/5885/castaway?fp=b-strategy
SWF URL: https://cache.armorgames.com/files/games/castaway-5885.swf?v=1373587523

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-09-12
Channel: nightly
Built: 2022-09-12T00:14:33.190Z
Commit: 474fd7fa8f74cabf40e7d091ac8e847dffd2814e
Is extension: false

# Metadata
width: 700
height: 500
frameRate: 30
numFrames: 5
swfVersion: 10
backgroundColor: #FFFFFF
isActionScript3: false
uncompressedLength: 7984791

**Comment by n0samu** [comment]
Different game, but this is the same error that's been happening with the Madness game: #7978

**Comment by n0samu** [comment]
Confirmed that this is still occurring even after #8248.

**Comment by n0samu** [comment]
Fixed by #8317
